TRLevelToolset/IOLogic/IOManager.cs

```diff
@@ -1,136 +1,129 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using TREnvironmentEditor;
-using TRLevelControl;
+﻿using TRLevelControl;
 using TRLevelControl.Model;
 
-namespace TRLevelToolset.IOLogic
+namespace TRLevelToolset.IOLogic;
+
+public static class IOManager
 {
-    public static class IOManager
-    {
-        public static TRLevelBase? CurrentLevel { get; set; }
+    public static TRLevelBase CurrentLevel { get; set; }
 
-        public static string? FileName { get; set; }
+    public static string FileName { get; set; }
 
-        public static TRGame LoadedGame { get; set; }
+    public static TRGame LoadedGame { get; set; }
 
-        public static TR1Level? CurrentLevelAsTR1 
+    public static TR1Level CurrentLevelAsTR1 
```

Comment:
Wait, why are we removing the `?`? This function can still return `null`.

Reply:
It's currently only called in the likes of the situation below, so we still check for null at that end. I find this approach cleaner, the VS warnings when `nullable` is on are less than helpful I find.

https://github.com/LostArtefacts/TR-Rando/blob/master/TRLevelToolset/Controls/DataControls/TR/TRAnimatedTextureControl.cs#L19

This is the UI when the level is null.

![image](https://github.com/LostArtefacts/TR-Rando/assets/33758420/7abe44f8-d067-4e3f-b61b-ff7b265b0e7a)

Reply:
I personally think it's cleaner to leave these intact. The warnings are not wrong, even if they're annoying.

Reply:
That's fair. I'll re-enable and look at clearing the warnings.